pom.xml

```diff
@@ -71,7 +71,7 @@
     </distributionManagement>
 
     <properties>
-        <oap-ws.project.version>21.1.4</oap-ws.project.version>
+        <oap-ws.project.version>21.1.5</oap-ws.project.version>
         <oap.deps.oap.version>21.1.2</oap.deps.oap.version>
```

Comment:
21.1.7 please

Reply:
Let's don't mix up separate tickets in my PR. It's your task to level up all xenoss repos, please
 do it in separate PR.